codecov.yml

```diff
@@ -0,0 +1,27 @@
+codecov:
+  require_ci_to_pass: no
+  notify:
+    wait_for_ci: no
+
+coverage:
+  precision: 2
+  round: down
+  range: "50...70"
+  status:
+    default_rules:
+      flag_coverage_not_uploaded_behavior: exclude
+    project:
+      target: auto
+      threshold: 0%
+      base: auto 
+      if_ci_failed: success
+      informational: false
+      only_pulls: true
+
+comment:
+  layout: "reach,diff,flags,files"
+  behavior: default
+  require_changes: no
+  require_base: no
+  require_head: no
+  after_n_builds: 1
```

Comment:
I could verify that this is correct using the doc. I am not sure about the rest of the changes.